### PR TITLE
PM with SafeMath

### PR DESCRIPTION
**Pull request by eladmallel**

### CODE REVIEW

**Comment by eladmallel** [comment]
Just note that this is branched off of https://github.com/livepeer/protocol/pull/269

File: contracts/pm/ERC20TicketBroker.sol [hunk 1]
```diff
@@ -2,9 +2,9 @@ pragma solidity ^0.4.25;
...
 pragma experimental ABIEncoderV2;
 
-import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
 import "./TicketBroker.sol";
-
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 
 // JUST AN EXAMPLE
 contract ERC20TicketBroker is TicketBroker {
```
Comment (yondonfu) [inline contracts/pm/ERC20TicketBroker.sol]: Missing a `using SafeMath for uint256`?

File: contracts/pm/TicketBroker.sol [hunk 2]
```diff
@@ -175,7 +176,7 @@ contract TicketBroker {
             sender.penaltyEscrow = 0;
         } else {
             amountToTransfer = _ticket.faceValue;
-            sender.deposit -= _ticket.faceValue;
+            sender.deposit = sender.deposit.sub(_ticket.faceValue);
```
Comment (yondonfu) [inline contracts/pm/TicketBroker.sol]: Due to the conditional on L171 (https://github.com/livepeer/protocol/pull/270/commits/db86d2bfb473523d8ada69ae2297bbe3ecf5df6d#diff-baed5767688fca63ab8b4efd937ce4ccR171) we know that this subtraction could never underflow so using SafeMath here just results in some extra unnecessary checks. I do see an argument for applying SafeMath everywhere for consistency sake. What do you think?

Comment (eladmallel) [reply contracts/pm/TicketBroker.sol]: I was prioritizing consistency :) If that's cool with you I'm happy to keep this change as-is. wdyt?

File: contracts/pm/ERC20TicketBroker.sol (hunk 1 above)
Comment (eladmallel) [reply contracts/pm/ERC20TicketBroker.sol]: Your EDIT is spot on. I removed the imports and using statements in subtypes.

File: contracts/pm/TicketBroker.sol (hunk 2 above)
Comment (yondonfu) [reply contracts/pm/TicketBroker.sol]: Consistency works for me!

**Pull request merged**